server/src/core/workspace/services/workspace.service.ts

```diff
@@ -161,7 +161,9 @@ export class WorkspaceService {
       this.delete({ where: { id: workspaceId } }),
     ]);
 
-    await this.tenantInitialisationService.delete(workspaceId);
+    try {
+      await this.tenantInitialisationService.delete(workspaceId);
```

Comment:
we should fix this method instead of try catch silently. Let's check if there is an existing workspace schema before trying to delete it

Reply:
we do that but we throw an error in that case. Maybe I can just return
![image](https://github.com/twentyhq/twenty/assets/29927851/2f519685-6e71-44dc-b5b4-1b4ca5f58e49)

Reply:
@martmull @charlesBochet I think this check is not necessary actually, your drop schema here will do a IF EXISTS so it should not fail